tools/roles/vllm/vars/main.yml

```diff
@@ -19,7 +19,21 @@ vllm_warning: "The vLLM container requires at least 60GB of storage. Additionall
 so please ensure that the necessary storage is available."
 installation_success_status: "vLLM succesfully installed"
 installation_failed_status: "Failed to install vLLM"
-vllm_python_version: "python3.9"
+
+# Validate inventory
+fail_no_node_kube_node: "Failed. vllm software is present in software_config.json. No node is part of kube_node group in inventory"
```

Comment:
vLLM

Reply:
vllm in smaller case what software name we supported in software_config.json